opm/simulators/flow/ActionHandler.cpp

```diff
@@ -237,9 +237,12 @@ applyActions(const int reportStep,
 
         logActiveAction(action->name(), matches.wells(), ts);
 
-        const auto wellpi = fetchWellPI<Scalar>
-            (reportStep, this->schedule_, this->wellModel_,
-             *action, matches, this->comm_);
+        // The std::unordered_map<std::string, Scalar> wellpi contains the well production indices from the last
+        // timestep. This map is needed for the keywords WELPI and WPIMULT.
```

Comment:
Strictly speaking, the current PI values aren't really needed for WPIMULT.  WPIMULT applies a constant CTF multiplier to (a subset of) the well's connections and the multiplier is given explicitly in the WPIMULT input data.  On the other hand, the WELPI keyword calculates the multiplier as a ratio of the input value and the current dynamic PI value.